api/cache/object_cache.go

```diff
@@ -0,0 +1,62 @@
+package cache
+
+import (
+	"time"
+
+	"github.com/bluele/gcache"
+	"github.com/nspcc-dev/neofs-api-go/pkg/object"
+)
+
+// ObjectsCache provides interface for lru cache for objects.
+type ObjectsCache interface {
+	Get(address *object.Address) *object.Object
+	Put(address *object.Address, obj object.Object) error
+	Delete(address *object.Address) bool
+}
+
+const (
+	// DefaultObjectsCacheLifetime is a default lifetime of objects in cache.
+	DefaultObjectsCacheLifetime = time.Minute * 5
+	// DefaultObjectsCacheSize is a default maximum number of objects in cache.
+	DefaultObjectsCacheSize = 1e6
+)
+
+type (
+	// ObjectHeadersCache contains cache with objects, lifetime of cache entries and client to establish connection with NeoFS.
```

Comment:
> client to establish connection with NeoFS

I don't see any client in `ObjectHeadersCache`

Reply:
Forgot to update the comment.